cmd/semantic-release/semantic-release.go

```diff
@@ -24,7 +24,7 @@ const (
 
 var (
 	// version is set at build time
-	Version  = "1.0.0"
+	Version  = "No version provided at build time"
```

Comment:
What happens if it is a fix, feature, or breaking change while the repository has no tags yet?
Will it keep the repository tags empty?

Reply:
@esequielvirtuoso I'm not one hundred percent sure, but I think this is semantic-release version, provided as a flag in the command line.

Seeing the code I saw now that the variable was not being used. I added `flag.StringVar()` to get it from command line and added it in `log.New()`.

As for a clean repo without tags, it will follow the logic in `getMostRecentTag` in **git.go**, [line 232](https://github.com/NeowayLabs/semantic-release/pull/13/files#diff-b06ea9cf8f1ecc1456ef02fbfbe397eba11dab36b113f74351134dce84208002R232), returning tag `0.0.0` :grin: 

Could you check the new changes?

Reply:
Everything looks good!